spotifython/playlist.py

```
@@ -1,228 +1,350 @@
         only one track is moved.
 
-        Parameters:
-        source_index:      An integer specifying the 0-indexed position of the first
-                           track to be moved. A negative integer will be evaluated 
-                           from the end of the playlist as negative indices behave in
-                           lists. This must be a valid index into a list of length
-                           len(playlist).
-
-        destination_index: An integer specifying the 0-indexed position of the track
-                           before which the other tracks will be moved. A negative
-                           integer will be evaluated from the end of the playlist as
-                           negative indices behave in lists. This must be a valid
-                           index into a list of length len(playlist).
-
-        Optional Parameters:
-        number:            A nonnegative integer specifying the number of tracks to
-                           be moved. Specifying 0 will result in no change to the
-                           playlist.
-
-        Returns:
-        A Response object.
+        Args:
+            source_index: An integer specifying the 0-indexed position of the
+                first track to be moved. A negative integer will be evaluated
+                from the end of the playlist as negative indices behave in
+                lists. This must be a valid index into a list of length
+                len(playlist).
+            destination_index: An integer specifying the 0-indexed position of
+                the track before which the other tracks will be moved. A
+                negative integer will be evaluated from the end of the playlist
+                as negative indices behave in lists. This must be a valid index
+                into a list of length len(playlist).
+            number: A nonnegative integer specifying the number of tracks to be
```

0 should raise an exception, probably programmer error

What if number > len(playlist)?